internal/redis/redis.go

```diff
@@ -536,8 +538,18 @@ func (c *DefaultConn) ZRange(key string, start, stop int) ([]string, error) {
 	return redis.Strings(c.conn.Do("ZRANGE", key, start, stop))
 }
 
-func (c *DefaultConn) ZRangeByScoreString(key string, stop int64) ([]string, error) {
-	return redis.Strings(c.conn.Do("ZRANGE", key, 0, stop, "BYSCORE"))
+func (c *DefaultConn) ZRangeByScoreString(key string, minScore int64, maxScore int64, count, offset int) ([]string, error) {
+	start := strconv.FormatInt(minScore, 10)
+	if minScore == 0 {
+		start = "-inf"
+	}
+	stop := strconv.FormatInt(maxScore, 10)
+	if maxScore == 0 {
+		stop = "+inf"
+	}
+
+	// range from start including start to stop excluding stop
+	return redis.Strings(c.conn.Do("ZRANGE", key, "("+start, stop, "BYSCORE", "LIMIT", offset, count))
```

Comment:
Is this missing a terminating `)`? Or is this just how you specify inclusive vs exclusive? Please add a comment if it's the latter.